api/src/business_ar_api/services/notification_service.py

```diff
@@ -130,9 +131,13 @@ def _get_pdfs(token: str, business: BusinessModel, filing: FilingModel) -> list:
         if filing.status == FilingModel.Status.COMPLETED.value:
             # add receipt pdf
             url = f'{current_app.config.get("PAY_API_URL")}/payment-requests/{filing.invoice_id}/receipts'
+            # format date & time of filing to pacific time
+            pacific_tz = timezone('America/Los_Angeles')
+            pacific_time_filing_date = filing.filing_date.astimezone(pacific_tz)
+            filing_date_time_formatted = pacific_time_filing_date.strftime("%B %d, %Y %I:%M %p Pacific Time")
```

Comment:
probably could use filing_date_time_iso = pacific_time_filing_date.isoformat() to make sure iso format date is used for apis

Reply:
The point of this is to have it **not in ISO format**.
This is what it looks like now:
![image](https://github.com/user-attachments/assets/83ccb89c-a166-4c9c-8137-868238f898ca)